docs/modules/project/pages/hacking.adoc

```diff
@@ -368,25 +368,33 @@ In addition, you can submit pull requests to downstream project:
 
 [%interactive]
 * [ ] submit a pull request downstream to update Asciidoctor reveal.js version in the Asciidoctor Docker Container
-** modify the `Dockerfile`, `Makefile` and `README.adoc` in: https://github.com/asciidoctor/docker-asciidoctor
+** modify the `Dockerfile`, `tests/asciidoctor.bats`, `README.md` and `README.adoc` in: https://github.com/asciidoctor/docker-asciidoctor
```

Comment:
I feel like this should be documented at https://github.com/asciidoctor/docker-asciidoctor "How to bump a version". What do you think @dduportal?

Reply:
Then we could simply link the related documentation.

Reply:
We can move forward while waiting for Damien input. Feel free to merge.

Reply:
Hello folks! That is, indeed, a good idea. Let me start a PR in the docker image PR for documenting it, and i'll link it in this doc afterwards.

Reply:
I've opened https://github.com/asciidoctor/docker-asciidoctor/pull/486 on the image side. May I have your review on it?

I'll open a PR here once merged on image side